bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/SingleProbeStrategy.scala

```diff
@@ -94,8 +95,10 @@ case class SingleProbeStrategy(key: AgsStrategyKey, params: SingleProbeStrategyP
       case (angle, st) => new CandidateValidator(params, mt, List(st)).exists(ctx.withPositionAngle(angle.toOldModel))
     }
 
-  override def queryConstraints(ctx: ObsContext, mt: MagnitudeTable): List[QueryConstraint] =
-    params.queryConstraints(ctx, mt).toList
+  override def queryConstraints(ctx: ObsContext, mt: MagnitudeTable): List[QueryConstraint] = Nil
```

Comment:
Right, I replaced `queryConstraints` by `catalogQueries` but didn't review the API
Will do on the next PR